dm/simulator/internal/mcp/mcp_test.go

```diff
@@ -0,0 +1,239 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mcp
+
+import (
+	"math/rand"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/pingcap/tiflow/dm/pkg/log"
+)
+
+type testMCPSuite struct {
+	suite.Suite
+	mcp *ModificationCandidatePool
+}
+
+func (s *testMCPSuite) SetupSuite() {
+	s.Require().Nil(log.InitLogger(&log.Config{}))
```

Comment:
If I understand correctly, by default, `s.Nil()` is actually `s.Assert().Nil()`.  I choose to use the `Require()` here because I expect the log initialization should finish before doing rest of the work.  

However, I can remove all those places using `s.Assert().XXX()` with `s.XXX()` so that the code is more concise.

Reply:
Sorry I forget it's Assert not Require.